### DISCUSSION TITLE
Retry a workflow with cURL

### DISCUSSION THREAD

**Discussion by sidify** [argoproj/argo-workflows#13525] [Q&A] [answered]
Hi I am trying to rerun an Argo Workflow through REST endpoint call. However it says it is Not implemented. Does anyone know how to get around this? And when would this be implemented? As per [apidocs][1] the below should be the endpoint.

e.g.

    curl https://localhost:2746/api/v1/workflows/argo/example-wf-template/retry --insecure
    {"code":12,"message":"Not Implemented"}

The Argo version I am using is: `v3.5.10`

Any insights would be helpful. I also do not want to use command line tools in the code.

  [1]: https://argo-workflows.readthedocs.io/en/latest/swagger/

**Comment by agilgur5** [comment, marked as answer]
The API method is PUT, but your cURL statement is sending a GET. You probably want to add `-X PUT` and perhaps some data in your request
